### PR TITLE
add .env.test to dotenv configuration, update readme

### PR DESCRIPTION
**Pull request by tatyanabortnik**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,47 +20,34 @@ URL = 'client domain' or 'http://localhost:5173'
 ```
 
 2. Create a database named `digestwell` in your PostgreSQL instance.
-3. You'll need to create a user manually in the database. Do a `POST` request to `http://localhost:3000/register` with the 
-following body: `{"email": "test@email.com", "password": "123", "firstName": "John", "lastName": "Doe"}`
+3. You'll need to create a user manually in the database. Do a `POST` request to `http://localhost:3000/register` with the
```
Comment (Wyna-7) [inline README.md:23]: do we still need this?

File: server/package.json [hunk 2]
```diff
@@ -3,7 +3,9 @@
   "version": "1.0.0",
   "main": "index.js",
   "scripts": {
-    "test": "jest",
+    "start": "node index.js",
+    "dev": "nodemon .",
+    "test": "ENV=test jest",
```
Comment (sebastiandve) [inline server/package.json:8]: niceee

**Review by Wyna-7 — CHANGES_REQUESTED** [review]

**Review by sebastiandve — APPROVED** [review]
very nice

File: README.md (hunk 1 above)
Comment (tatyanabortnik) [reply README.md:23]: removed

**Review by Wyna-7 — APPROVED** [review]

**Pull request merged**